### ISSUE TITLE
Network screen shows spinning wheel in status bar infinitely.

### ISSUE THREAD

**Issue by polina-c** [flutter/devtools#3965]
<img width="318" alt="Screen Shot 2022-04-07 at 9 43 22 AM" src="https://user-images.githubusercontent.com/12115586/162254807-90445664-62f2-4427-8178-acf2a09c791c.png">

**Comment by kenzieschmoll** [comment]
The spinning wheel is to indicate that we are recording network traffic, so as long as we are not paused, this will always be spinning. Perhaps there is a better way that we can indicate that we are listening for network traffic @InMatrix

**Comment by srawlins** [comment]
Agreed. Closing; if we are made aware of a better UI design, we can open an issue and implement it.
